css/style.css

```diff
@@ -1,10 +1,142 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
+ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+header {
+  position: fixed;
+  top: 0;
+  min-height: 75px;
+  padding: 0px 50px;
+  display: flex;
+  justify-content: flex;
+  align-items: center;
+  background-color: #ffffff;
+ 
+}
+@media (max-width: 500px) {
+  header {
+    flex-wrap: wrap;
+  }
+}
+
+  
+.logo {
+    margin-top: 15px;
+    width: 10%;
+    position: relative;
+  }
+
+
+.logo > img {
+  width: 100%;
+  height: 3.375rem;
+  max-width: 200px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  text-align: center;
+  margin-left: 20px;
+}
+
+.nav-bar{
+  background-color: #ffffff;
+  height: 3.375rem;
+  width: 70rem;
+	padding: 0 0.938rem;
+	margin: 0 auto;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 500;
+  
+  
+  
+}
+
+nav > ul {
+  width: 35vw;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  float: right;
+  font-family: 'Roboto', sans-serif;
+  font-size: 1em;
+  letter-spacing: 0.03rem;
+	margin-top: 1.563rem;
+  color: #838994;
+
+}
+li {
+  list-style: none;
+}
+
+li:hover {
+	cursor: pointer;
+	color: #333333;
+}
+
+
+.introduction  {
+    background-image: url("http://swarley00.github.io/startup-landing-page-clone/images/first-background.jpg");
```

Comment:
Why not link to the image in your img folder?

Reply:
It works in the same way: `url('./img/image-name.jpg')`

Note: This is just an example and may not be the exact path.

Reply:
many times told me path did not exist.